GObject-2.0.gir

```diff
@@ -14376,13 +14396,13 @@ by using g_object_unref().</doc>
           <type name="Object" c:type="gpointer"/>
         </return-value>
         <parameters>
-          <instance-parameter name="weak_ref" direction="inout" caller-allocates="0" transfer-ownership="full">
```

Comment:
Checking if this is intentional

Reply:
as we don't generate bindings of this, let us merge it already